dali/pipeline/data/views.h

```diff
@@ -145,12 +149,16 @@ view(const Tensor<Backend> &data) {
 template <typename T, int ndim = kernels::DynamicDimensions, typename Backend>
 kernels::TensorView<detail::storage_tag_map_t<Backend>, T, ndim>
 view_as_tensor(const Tensor<Backend> &data) {
+  static_assert(std::is_const<T>::value,
+                "Cannot create a non-const view of a `const Tensor<>`. Missing `const` in T?");  // NOLINT
```

Comment:
I think it's not necessary. The underlying function will raise the same assert.

Reply:
Done